generator/internal/gapic_utils.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google Inc.  All rights reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "gapic_utils.h"
+
+namespace google {
+namespace api {
+namespace codegen {
+namespace internal {
+
+std::string LocalInclude(std::string header) {
+  header.insert(header.begin(), '"');
```

Comment:
It's deceptively safe to mutate the string here since it's being passed and returned by value, but will change to absl methods to maintain consistency.